### PR TITLE
Смирнов -- Лабораторная работа #0

### PR DESCRIPTION
**Pull request by SmirnovEgorRu**

### CODE REVIEW

**Comment by SmirnovEgorRu** [comment]
@vgvoleg. @PetrovKP, please, review.

File: code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/EgorSmirnovTest.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import ru.unn.agile.ComplexNumber.model.ComplexNumber;
+import static org.junit.Assert.*;
+
+public class EgorSmirnovTest {
+    @Test
+    public void canAddNumbers() {
+        var number1 = new ComplexNumber(5.1, 10.1);
+        var number2 = new ComplexNumber(-5.1, -10.1);
+
+        var product = number1.add(number2);
```
Comment (PetrovKP) [inline code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/EgorSmirnovTest.java]: product -> sum ?

Comment (SmirnovEgorRu) [reply code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/EgorSmirnovTest.java]: @PetrovKP Applied

**Comment by PetrovKP** [comment]
👍

**Comment by SmirnovEgorRu** [comment]
@Lenferd, @PovelikinRostislav, could you, please, have a look?

**Comment by KalyanovDO** [comment]
:+1:

**Comment by SmirnovEgorRu** [comment]
@kirill-korniakov

**Pull request merged**